tests/conftest.py

```diff
@@ -0,0 +1,134 @@
+"""
+__author__: HashTagML
+license: MIT
+Created: Thursday, 1st January 1970
+"""
+
+from pathlib import Path
+from typing import Union
+import pytest
+import numpy as np
+import pandas as pd
+
+import os
+import shutil
+
+from kaggle.api.kaggle_api_extended import KaggleApi
+
```

Comment:
I think we need to check `tensorflow` installation, before expecting it while running test case